app/controllers/users_controller.rb

```diff
@@ -0,0 +1,41 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+  def show
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to user_path(@user)
+    else
+      render :new
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    @user.update(user_params)
+    redirect_to user_path(@user)
+  end
+
+  def destroy
+    @user.destroy
+    redirect_to root_path
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit(:first_name, :last_name, :email, :phone_number)
+  end
+
+  def set_cocktail
```

Comment:
Change method name to set_user
`def set_user`
`@user = User.find(params[:id])`
`end`

Reply:
Why do we need a set method inside the user controller, isn't that only needed inside reviews

Reply:
It helps refactor the code against repetition of the .find method